presto-main/src/main/java/com/facebook/presto/sql/analyzer/Analysis.java

```diff
@@ -496,6 +500,15 @@ public void registerNamedQuery(Table tableReference, Query query)
         namedQueries.put(tableReference, query);
     }
 
+    public void registerTableForView(Table tableReference) {
```

Comment:
This formatting is not correct. You can install the code formatter for IntelliJ which will fix this automatically (when you run the formatter): https://github.com/airlift/codestyle

This specific error is caught by Checkstyle, which causes the build to fail. You can see the Travis build result for this Pull Request.